tests/test_node.py

```diff
@@ -48,8 +49,8 @@ def test_child_by_field_id(self):
         fn_node = tree.root_node.children[0]
 
         self.assertIsNone(self.python.field_id_for_name("noname"))
-        name_field = self.python.field_id_for_name("name")
-        alias_field = self.python.field_id_for_name("alias")
+        name_field = cast(int, self.python.field_id_for_name("name"))
+        alias_field = cast(int, self.python.field_id_for_name("alias"))
         self.assertIsNone(root_node.child_by_field_id(alias_field))
         self.assertIsNone(root_node.child_by_field_id(name_field))
         self.assertIsNone(fn_node.child_by_field_id(alias_field))
```

Comment:
>_possible bug_

**✨ AI Suggestion:** The cast to `int` might be unnecessary as `field_id_for_name` already returns an integer or None. Consider removing the cast or handling the `None` case explicitly.
```suggestion
        name_field = self.python.field_id_for_name("name")
        alias_field = self.python.field_id_for_name("alias")
        self.assertIsNone(root_node.child_by_field_id(alias_field))
        self.assertIsNone(root_node.child_by_field_id(name_field))
        self.assertIsNotNone(fn_node.child_by_field_id(name_field))
```